config/initializers/secure_headers.rb

```diff
@@ -35,21 +35,17 @@
     ],
     style_src: ["'self'", IdentityConfig.store.asset_host.presence],
     base_uri: ["'self'"],
+    disable_nonce_backwards_compatibility: true,
```

Comment:
should we feature flag this in case we have it deployed and get feedback it's broken?

Reply:
Yeah, we who knows where we have js / css without a nonce